Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts/Fitnet.Contracts.csproj

```diff
@@ -7,6 +7,7 @@
     <ItemGroup>
         <PackageReference Include="EvolutionaryArchitecture.Fitnet.Common.Api" Version="1.2.1" />
         <PackageReference Include="JetBrains.Annotations" Version="2023.3.0" />
+        <PackageReference Include="Microsoft.FeatureManagement" Version="3.2.0" />
```

Comment:
Is code from this PR related to this change?

Reply:
By accident :) Removed